src/Extensions/FileFormFactoryExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SilverStripe\CampaignAdmin\Extensions;
+
+use SilverStripe\AssetAdmin\Forms\FileFormFactory;
+use SilverStripe\Assets\File;
+use SilverStripe\Core\Extension;
+use SilverStripe\Forms\FormAction;
+use SilverStripe\Security\Permission;
+
+/**
+ * Extension that updates the Popover menu of `FileFormFactory`.
+ * This extension will only be applied if the `campaign-admin` module is installed.
```

Comment:
```suggestion
```
And remove similar wording from other copied code, if it's there

Reply:
Removed. Also removed from AddToCampaignExtension which had the same comment